radixdlt-core/radixdlt/src/main/java/com/radixdlt/network/transport/tcp/NettyTCPTransportImpl.java

```diff
@@ -109,6 +122,7 @@
 			TCPConstants.METADATA_PORT, String.valueOf(port)
 		);
 		this.priority = config.priority(0);
+		this.messageBufferSize = config.messageBufferSize(1);
```

Comment:
This default value of 1 seems way too small.  It should be some multiple of the number of validators, so we don't lose vote messages.

Reply:
This is a per channel buffer.